### PR TITLE
Configure Travis to deploy new docker images to Docker hub every merge to master

### PR DESCRIPTION
**Pull request by bwillard**

### CODE REVIEW

File: docker_push.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+#Script used to push new docker images to docker hug by travis.
+
+./gradlew -PcloudType=local :distributions:demo-server:dockerize
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
```
Comment (rtannenbaum) [inline docker_push.sh]: can you document how these env variables are set?

**Comment by holachuy** [comment]
Does push to docker hub mean each commit is a published version, can we distinguish between latest and an RC or prod version?

Just hate for folks to pull and run latest when it could be broken.

File: docker_push.sh (hunk 1 above)
Comment (bwillard) [reply docker_push.sh]: Done.

**Comment by rtannenbaum** [comment]
@bwillard I think we need to distinguish between push/commit and merge though, for example a PR can contain 10 commits but only 1 merge into master. Is it possible to make it only build the new image on merge PR to master? We probably don't need it for every commit

**Comment by bwillard** [comment]
@rtannenbaum sorry, yes that is what this does is only run the docker push on the merge to master, not on every commit.

**Comment by rtannenbaum** [comment]
Ah perfect, thanks. Can you update PR description? LGTM

**Pull request merged**